Tests/Jupyter/QuantBookIndicatorsTests.cs

```diff
@@ -16,20 +16,21 @@
 using NUnit.Framework;
 using Python.Runtime;
 using System;
-using System.IO;
+using QuantConnect.Securities;
 
 namespace QuantConnect.Tests.Jupyter
 {
-    [TestFixture, Category("TravisExclude")]
+    [TestFixture]
     public class QuantBookIndicatorsTests
     {
         dynamic _module;
 
         [TestFixtureSetUp]
         public void Setup()
         {
-            var pythonPath = new DirectoryInfo("Jupyter/RegressionScripts");
-            Environment.SetEnvironmentVariable("PYTHONPATH", pythonPath.FullName);
+            SymbolCache.Clear();
```

Comment:
not for now, but next time, I think there may be a way to do a global test clean up function, after each test we should be resetting all static state so the test starts clean